src/index.jsx

```diff
@@ -9,10 +9,15 @@ import thunk from 'redux-thunk';
 
```

Comment:
Hmm I see we are using redux-thunk here. Why would't we stay with redux-saga? I personally think it's a better library to handle async actions.